docs/fundamentals/database-collection.txt

```diff
@@ -161,17 +161,107 @@ as in the preceding example, but the query is constructed by using the
+about the collections in a database:
 
-The following example accesses a database connection, then
-calls the ``listCollections()`` method to retrieve information about the
-collections in the database:
+- :ref:`laravel-list-coll-command`
+- :ref:`laravel-list-coll-methods`
+
+.. _laravel-list-coll-command:
+
+Run a Shell Command
+~~~~~~~~~~~~~~~~~~~
+
+You can list the collections in a database by running the following
+command in your shell from your project's root directory:
+
+.. code-block:: bash
+
+   php artisan db:show
+
+This command outputs information about the configured database and lists its
+collections under the ``Table`` header. For more information about the ``db:show``
+command, see `Laravel: New DB Commands <https://blog.laravel.com/laravel-new-db-commands-and-more>`__
+on the official Laravel blog.
+
+.. _laravel-list-coll-methods:
+
+Call Database or Schema Methods
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can list the collections in a database by calling the following
+methods in your application:
+
+- ``DB::listCollections()``: lists information about each collection by
+  using the query builder
+- ``Schema::getTablesListing()``: lists the name of each collection by
+  using the schema builder
+- ``Schema::getTables()``: lists the name and size of each collection by
+  using the schema builder
+
+Example
+```````
+
+The following example accesses a database connection, then calls the
+``listCollections()`` query builder method to retrieve information about
+the collections in the database:
 
 .. code-block:: php
 
    $collections = DB::connection('mongodb')->getMongoDB()->listCollections();
 
+List Collection Fields
+----------------------
+
+You can take either of the following actions to see information
+about each field in a collection:
+
+- :ref:`laravel-list-fields-command`
+- :ref:`laravel-list-fields-methods`
+
+.. _laravel-list-fields-command:
+
+Run a Shell Command
+~~~~~~~~~~~~~~~~~~~
+
+You can see a list of fields in a collection by running the following
+command in your shell from your project's root directory:
+
+.. code-block:: bash
+
+   php artisan db:table <collection name>
+
+This command outputs each collection field and its corresponding data type
+under the ``Column`` header. For more information about the ``db:table``
+command, see `Laravel: New DB Commands <https://blog.laravel.com/laravel-new-db-commands-and-more>`__
+on the official Laravel blog.
+
+.. _laravel-list-fields-methods:
+
+Call Schema Methods
+~~~~~~~~~~~~~~~~~~~
+
+You can list the fields in a collection by calling the ``Schema::getColumns()``
+schema builder method in your application.
+
+You can also use the following methods to return more information about the
+collection fields:
+
+- ``Schema::hasColumn(string $<field name>)``: checks if the specified field exists
+  in at least one document
+- ``Schema::hasColumns(string[] $<field names>)``: checks if each specified field exists
+  in at least one document
```

Comment:
This need to mention that MongoDB is a schemaless database, so this functions checks the data, not the schema like in traditional databases. If the collection doesn't exist or is empty, this function always return `false`.

Reply:
I added a note clarifying this, let me know what you think